### PR TITLE
Send error email on poster generation failure

### PR DESCRIPTION
**Pull request by ibarrae**

### CODE REVIEW

File: api/posters/start/start.utils.tsx [hunk 1]
```diff
@@ -581,3 +582,83 @@ export async function sendUpdateToClient(
...
+        backgroundColor: 'black',
+        margin: '0 auto',
+        padding: 5,
+        width: 'max-content',
+      }}
+    >
+      <Logo aria-hidden width={89} height={32} />
+    </div>
+  </div>
+)
```
Comment (JoseRFelix) [inline api/posters/start/start.utils.tsx:664]: I think this should be in the components folder of the API.

File: api/posters/start/start.utils.tsx [hunk 2]
```diff
@@ -581,3 +582,83 @@ export async function sendUpdateToClient(
...
+      },
+      Body: {
+        Html: {
+          Data: ReactDOMServer.renderToString(<ErrorNotification />),
+        },
+      },
+    },
+  }
+
+  logger.info('Sending error notification to user...')
```
Comment (JoseRFelix) [inline api/posters/start/start.utils.tsx:616]: :+1:

File: api/posters/start/start.utils.tsx [hunk 3]
```diff
@@ -581,3 +582,83 @@ export async function sendUpdateToClient(
...
+      >
+        4
+      </span>
+    </p>
+    <div
+      style={{
+        backgroundColor: 'black',
+        margin: '0 auto',
+        padding: 5,
+        width: 'max-content',
```
Comment (JoseRFelix) [inline api/posters/start/start.utils.tsx:658]: Is `width: max-content` available in email clients?

**Review by JoseRFelix — CHANGES_REQUESTED** [review]
Looking good! Left some comments. Let me know what do you think

File: api/posters/start/start.utils.tsx (hunk 3 above)
Comment (ibarrae) [reply api/posters/start/start.utils.tsx:658]: Oh yeah, you're right, I think this needs to be updated then. Great catch :tada:

**Review by JoseRFelix — APPROVED** [review]

**Pull request merged**